runtime/debian.rules

```diff
@@ -20,7 +20,4 @@ override_dh_auto_build:
 
 override_dh_auto_install:
 	mkdir -p debian/cc-runtime
-	make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr
-
-
-
+	cd $(GOPATH)/src/github.com/clearcontainers/runtime/ && make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr SYSCONFDIR=/etc
```

Comment:
For some reason, during the ubuntu build, it does not work having it splitted in two lines. I guess is related to the makefile stuff